### DISCUSSION TITLE
Document --react-native-version flag in README

### DISCUSSION THREAD

**Discussion by obasille** [callstack/react-native-builder-bob#441] [Ideas]
I found about this flag by looking at the source code. I think it would be valuable to document it in the README as not everyone is running on the latest version of React Native.

Thanks for making this great package!

**Comment by atlj** [comment]
Hey, sorry for the late reply. The problem with this argument is that we don't support it thoroughly for the moment. If you try to create a library using `npx create-react-native-library` and specify an older version of React Native such as `0.62`, there is no guarantee it would work. So it makes sense to keep it 'hidden' for now.

However, we have plans to document all the supported versions of React Native that are compatible with `create-react-native-library`. We can mention this argument once the documentation for React Native versions is out.
